account_view_adj/views/account_invoice_views.xml

```diff
@@ -5,7 +5,7 @@
             <field name="model">account.invoice</field>
             <field name="inherit_id" ref="account.invoice_supplier_tree"/>
             <field name="arch" type="xml">
-                <xpath expr="//tree/field[@name='number']" position="after">
+                <xpath expr="//tree/field[@name='is_invoice_issuer']" position="before">
```

Comment:
You can't do this as `is_invoice_issuer` comes from a custom module. Use a field from `account` module as the hook point.

Reply:
I changed the position of "Responsible" to before the` 'date_due'.`